src/commonMain/kotlin/Foxy.kt

```diff
@@ -79,7 +79,9 @@ object Foxy {
      * @return Whether the existing session was valid.
      */
     fun authenticateExistingSession(): Boolean {
-        val (token, date, integrity, _) = tokenStorageGet().split(";")
+        val storedToken = tokenStorageGet()
+        if (storedToken == "") return false
```

Comment:
I think we should have it written as the following to be more clear as to what we're checking:

```kotlin
if (storedToken.isEmpty())
    return false
```